docs/source/examples/managed-jobs.rst

```diff
@@ -535,8 +545,8 @@ To achieve the above, you can specify custom configs in :code:`~/.sky/config.yam
         # Specify the location of the jobs controller.
         cloud: gcp
         region: us-central1
-        # Specify the maximum number of managed jobs that can be run concurrently.
-        cpus: 4+  # number of vCPUs, max concurrent jobs = 2 * cpus
+        # Allow more managed jobs to be run concurrently. (Default: 4+)
```

Comment:
```suggestion
        # Bump this to allow more managed jobs to be run concurrently. (Default: 4+)
```

Reply:
Actually, should we recommend memory here as well given the new changes?

Reply:
By default memory is 8x so it will scale with CPU, but we can add a line and a comment about it too.